common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -13,7 +13,7 @@
  * the License.
  */
 
-package tachyon.client;
+package tachyon.worker;
```

Comment:
this should either be just tachyon or the class should be called just NetAddress. having tachyon.worker.WorkerNetAddress is redundant

(a side benefit of doing tachyon.worker.NetAddress is that there won't be a name conflict with tachyon.thrift.WorkerNetAddress)